packages/loopback/test/acceptance/module-exporting/feature.md

```diff
@@ -0,0 +1,18 @@
+# Feature: Module exporting
+
+- In order to use the LoopBack.next module
+- As an app developer
+- I want to import the module
+- So that I can use LoopBack.next framework in my project
+
+## Scenario: JavaScript project using Node.js's require syntax
+
+- Given a Node.js project
+- And LoopBack.next installed as a project dependency
+- When I import LoopBack.next's Application construct via Node.js's require syntax
+- Then I get an instantiable Application object
+
+```js
+const Application = require('@loopback/loopback').Application;
+const app = new Application();
+```
```

Comment:
Ignore these, I'm branching off another branch that has these files. Will disappear once I land https://github.com/strongloop/loopback-next/pull/213 and rebase.